Makefile

```diff
@@ -95,26 +101,26 @@ $(GEN)/connect-core: node_modules/.bin/protoc-gen-es packages/connect-core/buf.g
 	@mkdir -p $(@D)
 	@touch $(@)
 
-$(GEN)/connect-web-test: node_modules/.bin/protoc-gen-es $(BUILD)/protoc-gen-connect-web packages/connect-web-test/buf.gen.yaml Makefile
+$(GEN)/connect-web-test: node_modules/.bin/protoc-gen-es $(BUILD)/protoc-gen-connect-web $(BUILD)/protoc-gen-connect-es packages/connect-web-test/buf.gen.yaml Makefile
```

Comment:
@timostamm I added this alongside the connect-web plugin because my thought was we can just go and remove it at some point without having to remove and then add.  Lmk if you'd rather it be a different configuration.

Reply:
No buf.gen.yaml uses `protoc-gen-connect-web` anymore, right? 

We still want to be able to make bugfixes, so we should keep it in `build` and in `release` for now.

But can't we remove it here and at other places?

Reply:
Yeah, good call.  Now that it's removed from those `buf.gen.yaml` files, it's not needed in the individual package pre-reqs any more.  Removed.